src/gui/PianoView.cpp

```diff
@@ -618,6 +619,9 @@ void PianoView::mouseMoveEvent( QMouseEvent * _me )
  */
 void PianoView::keyPressEvent( QKeyEvent * _ke )
 {
+#ifdef LMMS_BUILD_LINUX
```

Comment:
> To leave the possibility to use another key for something else:-)

Well if we have to hard-code individual key codes to obtain this feature, I'd argue we're doing something wrong here.  How about we use the virtual key for shift and move on.